### ISSUE TITLE
TypeError: Cannot create property 'Symbol(reported)' on string '...'

### ISSUE THREAD

**Issue by simllll** [elastic/apm-agent-nodejs#509]
in lib/instrumentation/modules/express.js there is following error handler check:
```
if ((isError(err) || typeof err === 'string') && !err[reportedSymbol]) {
            err[reportedSymbol] = true
            agent.captureError(err, { request: req })
}
```
there is a check if it is a string, in strict mode a string cannot have a property though. So "err[reportedSymbol] = true" does fail in this case with the error "Cannot create property 'Symbol(reported)' on string '...'."

Not quite sure how to deal with this, I'm not totally aware why this flag is needed at all? Why would it be called twice?
Alternatively it would be also great to deactivate this error handling when captureExceptions is set to false.

Thanks
Simon

**Comment by Qard** [comment]
What Node.js version are you running? It doesn't seem to be failing for me. 🤔

**Comment by simllll** [comment]
It's only happening in strict mode. I'm running node 8, but as far as I can say this behaviour is happening in all node versions.

Just try
```
"use strict";

var a = 'hello';
a.something = true;
```

Regards
Simon

**Comment by watson** [comment]
I can confirm this is a bug 👍

**Comment by watson** [comment]
Fixed and released in version 1.11.0. Thanks @simllll for fixing this bug 😃
